libPhoneNumber-Demo/libPhoneNumber-Demo/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Maybe we can use the icon from here: https://opensource.google/projects/libphonenumber

Reply:
Yup, got the image and created the icons.